1_web/1-1_track_hugo/README.md

```diff
@@ -1 +1,172 @@
-# 1.1 - Track Hugo
\ No newline at end of file
+# 1.1 - Track Hugo
+## Introduction
+
+Welcome to the Hugo Portfolio Lab! In this lab, you'll create a portfolio website using Hugo, a powerful and flexible static site generator. This portfolio will not only serve as a showcase of your skills and projects but also lay the groundwork for your onboarding journey with the club. This lab will guide you through the process of setting up Hugo, choosing a template, and ultimately building a website that represents your professional self.
+
+If you haven't read the formal introduction to this onboarding series, we recommend starting there. If you're ready and have chosen Hugo as your web development framework, let's proceed!
+
+This lab is structured to accommodate different levels of engagement:
+
+- **Green (Essentials):** 🟢 Read and complete the essential parts to set up your basic website.
+- **Yellow (Intermediate):** 🟡 Learn about more advanced concepts and make small customizations.
+- **Red (Advanced):** 🔴 Dive deep into Hugo and customize every part of your website.
+
+No matter your level of interest, you'll leave this lab with a working portfolio. The only difference is how deep you decide to explore the Hugo ecosystem.
+
+## Lab Structure
+
+### 1. Getting Started with Hugo
+
+#### 🟢 Introduction to Hugo
+
+- **What is Hugo?**: Hugo is an open-source static site generator designed for speed and simplicity. It is highly suitable for creating lightweight, fast-loading websites.
+- **Why Hugo?**: Hugo is renowned for its speed and ease of use, thanks to its foundation in Go. It allows developers to create fast, secure websites by quickly transforming Markdown-based content into static HTML files. With its simple setup, customizable templates, and instant builds, Hugo streamlines the website development process—ideal for blogs, documentation, and any content-driven site.
+
+#### 🟢 Installing Hugo
+
+- **Option 1: Using Docker** (Recommended for Beginners)
+  - **Why Docker?**: Docker is a better choice, as it simplifies setup, ensures consistency, and helps build foundational knowledge for later labs. Docker will aslo be used to ultimately deploy your website on the club's infrastructure.
+    - Here is a link to the club’s ["Learn Docker" wiki article](https://wiki.omni.cedille.club/onboarding/tracks/learn-docker/) for first timers.
+  - **Installation Steps**: Explain the steps to pull an image from the docker hub and run it.
```

Comment:
Here we want to explain the steps to pull and run it. We are not supposed to keep 'Explain the steps to pull an image from the docker hub and run it.', we litterally have to explain like what that phrase says. 

I propose that for every command that the track gives to run, the alternative we give is that the user can run 
`docker run --rm -it -v $(pwd):/src -p 1313:1313 floryn90/hugo:0.135.0 yourcommand` which is the equivalent to `hugo version`
e.g.: `docker run --rm -it -v $(pwd):/src -p 1313:1313 floryn90/hugo:0.135.0 version`